extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -156,7 +156,7 @@ export class AmpIframe extends AMP.BaseElement {
         'amphtml/blob/master/spec/amp-iframe-origin-policy.md for details.',
         element);
     userAssert(!(endsWith(hostname, `.${urls.thirdPartyFrameHost}`) ||
-        endsWith(hostname, '.ampproject.org')),
+        endsWith(hostname, '.amp.dev')),
```

Comment:
I don't think that his ampproject.org should be replaced here as this has nothing to do with the documentation.

Reply:
fixed